tus_storagehandler/app.py

```diff
@@ -0,0 +1,57 @@
+"""API server entry point."""
+
+import logging
+import os
+from pathlib import Path
+
+from connexion import FlaskApp  # type: ignore
+from dotenv import load_dotenv  # type: ignore
+from foca import Foca
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+
+
+def init_app() -> FlaskApp:
+    """Initialize and return the FOCA app.
+
+    This function initializes the FOCA app by loading the configuration
+    from the environment variable `TUS_FOCA_CONFIG_PATH` if set, or from
+    the default path if not. It raises a `FileNotFoundError` if the
+    configuration file is not found.
+
+    Returns:
+        A Connexion application instance.
+
+    Raises:
+        FileNotFoundError: If the configuration file is not found.
+    """
+    # Determine the configuration path
+    config_path_env1 = os.getenv("TUS_FOCA_CONFIG_PATH")
+    print(config_path_env1)
+    if config_path_env := os.getenv("TUS_FOCA_CONFIG_PATH"):
+        print(config_path_env)
+        config_path = Path(config_path_env).resolve()
+    else:
+        config_path = (
+            Path(__file__).parents[1] / "deployment" / "config.yaml"
+        ).resolve()
+
+    # Check if the configuration file exists
+    if not config_path.exists():
+        raise FileNotFoundError(f"Config file not found at: {config_path}")
+
+    foca = Foca(
+        config_file=config_path,
+    )
+    return foca.create_app()
+
+
+def main() -> None:
+    """Run FOCA application."""
+    app = init_app()
+    app.run(port=app.port)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
One thing which I am planing on doing is setting a `dev` and `prod` env in tesk so that if I run `python tesk/app.py` (this will invoke the main method from the if condition) it should run on the port 8081 and when I run the server with my console script `tesk` (which mean I am running in prod, this will directly invoke the main method) it should run on the default port ie, 8080. The benifit is lets say you wanna run a stable version of this application (viq `docker compose` or `helm` etc) and development version (via `python tus_storagehandler/app.py` ) to compare the results or see what broke you don't have to portforward to diffrent ports.

If you think this is something you might find useful, I suggest you do so, here is how I would do that:

```python
def main(env: str = None) -> None:
    """Run FOCA application.

    Args:
        env (str, optional): The environment in which to run the application.
                             Defaults to 'prod' if not specified.
    """
    # Set default port
    default_port = 8081

    # Initialize application
    app = init_app()

    # Set environment and port
    if env is None:
        env = 'prod'
    port = app.port if env == 'prod' else default_port

    # Run the application
    app.run(port=port)


if __name__ == "__main__":
    # Get environment variable or default to 'dev'
    environment = os.getenv('ENV', 'dev')
    main(environment)
```

Though I think this kinda thing should be handled by FOCA itself but I don't think we should wait because its a must have for testing so IMO you should do it too. Thoughts @uniqueg?